### PR TITLE
Login dialog done HTML page

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-access/0.1/amp-login-done-dialog.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+   * the following form:
+   * ```
+   * [lang="fr"], [lang="fr-FR"] {display: block}
+   * ```
+   * @return {string}
+   * @private
+   */
+  buildStyles_() {
+    const nav = this.win.navigator;
+    const lang = nav.language || nav.userLanguage || 'en-US';
```
Comment (cramforce) [inline extensions/amp-access/0.1/amp-login-done-dialog.js]: Neat. Probably makes sense to prefer a URL arg like "?hl=de", so pubs can chose to tell us the lang of the transaction.

File: gulpfile.js [hunk 2]
```diff
@@ -527,6 +528,79 @@ function buildExperiments(options) {
 
 
 /**
+ * Build "Login Done" page.
```
Comment (cramforce) [inline gulpfile.js]: Please put in new file. We try to not make this bigger.

File: extensions/amp-access/0.1/amp-login-done-dialog.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  /**
+   * The language is selected based on `navigate.language` by setting CSS of
+   * the following form:
+   * ```
+   * [lang="fr"], [lang="fr-FR"] {display: block}
+   * ```
+   * @return {string}
+   * @private
+   */
+  buildStyles_() {
```
Comment (cramforce) [inline extensions/amp-access/0.1/amp-login-done-dialog.js]: I'd recommend factoring out the lang section into a stateless function (takes args, returns value) and then unit test that separately.

File: extensions/amp-access/0.1/amp-login-done-dialog.js (hunk 1 above)
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-login-done-dialog.js]: Ok. I'll add the attribute and default to `navigator.language` when not specified.

File: gulpfile.js (hunk 2 above)
Comment (dvoytenko) [inline gulpfile.js]: I can't yet - there's task somewhere in @erwinmombay 's queue to make it possible with compilation targets.

File: extensions/amp-access/0.1/amp-login-done-dialog.js (hunk 3 above)
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-login-done-dialog.js]: Yes, it's unit-tested separately.

File: gulpfile.js (hunk 2 above)
Comment (cramforce) [inline gulpfile.js]: Can't you just put it in some file and export the functions?

File: extensions/amp-access/0.1/amp-login-done-dialog.js (hunk 3 above)
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-login-done-dialog.js]: Done.

**Comment by cramforce** [comment]
LGTM

File: gulpfile.js (hunk 2 above)
Comment (dvoytenko) [inline gulpfile.js]: Per @erwinmombay "right now it's kind of hard".

**Comment by dvoytenko** [comment]
I added "hl" query parameter and also added additional fallback when no nodes match the selection.

File: gulpfile.js (hunk 2 above)
Comment (erwinmombay) [inline gulpfile.js]: probably wouldn't be `too hard` but wouldn't want you to waste your time on it on this PR. I'll track and work on it. https://github.com/ampproject/amphtml/issues/1346

**Pull request merged**